package.json

```diff
@@ -68,56 +68,56 @@
 		}
 	},
 	"dependencies": {
-		"@babel/core": "^7.12.10",
-		"@babel/plugin-proposal-class-properties": "7.12.1",
+		"@babel/core": "^7.24.6",
 		"@babel/plugin-syntax-import-meta": "^7.10.4",
-		"@babel/plugin-syntax-jsx": "^7.12.1",
-		"@babel/plugin-transform-flow-strip-types": "^7.12.10",
-		"@babel/plugin-transform-react-jsx": "^7.12.11",
-		"@babel/plugin-transform-regenerator": "^7.12.1",
-		"@babel/preset-env": "^7.12.11",
-		"@babel/preset-flow": "^7.12.1",
-		"@babel/preset-react": "^7.12.10",
-		"@rollup/plugin-alias": "^3.1.1",
-		"@rollup/plugin-babel": "^5.2.2",
-		"@rollup/plugin-commonjs": "^17.0.0",
+		"@babel/plugin-syntax-jsx": "^7.24.6",
+		"@babel/plugin-transform-class-properties": "7.24.6",
+		"@babel/plugin-transform-flow-strip-types": "^7.24.6",
+		"@babel/plugin-transform-react-jsx": "^7.24.6",
+		"@babel/plugin-transform-regenerator": "7.17.9",
```

Comment:
TODO: somewhere after this results in regenerator ending up in the bundles, blowing them up rather significantly.

Reply:
Oh nice, `7.18.0` ended up inlining regenerator as a normal helper, dropping the global: https://github.com/babel/babel/pull/14538

This is a sizable increase but IMO it's completely reasonable.